### ISSUE TITLE
Docs development build fails

### ISSUE THREAD

**Issue by VanTanev** [davidkpiano/xstate#2101]
**Description**
Trying to run `yarn docs:dev` produces the following error:
```
➜  xstate git:(master) ✗ yarn docs:dev
yarn run v1.21.1
$ npm run docs:packages && cd docs && vuepress dev .
npm WARN lifecycle The node binary used for scripts is /tmp/yarn--1618483255509-0.8178695867708952/node but npm is using /home/ivan/.nvm/versions/node/v14.15.5/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> xstate-monorepo@1.19.1 docs:packages /home/ivan/dev/xstate
> cpy 'packages/*/README.md' docs --parents --rename=index.md && cpy 'packages/*/docs/*.md' docs --parents

/home/ivan/dev/xstate/node_modules/vue-server-renderer/index.js:8
  throw new Error(
  ^

Error: 

Vue packages version mismatch:

- vue@3.0.7
- vue-server-renderer@2.6.12

This may cause things to work incorrectly. Make sure to use the same version for both.

    at Object.<anonymous> (/home/ivan/dev/xstate/node_modules/vue-server-renderer/index.js:8:9)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/ivan/dev/xstate/node_modules/@vuepress/core/lib/node/build/index.js:11:34)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/ivan/dev/xstate/node_modules/@vuepress/core/lib/node/App.js:21:22)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected Result**
`yarn docs:dev` should work.

**Actual Result**
Cannot work on docs locally.

**Additional context**
```
➜  xstate git:(master) ✗ git log --oneline -1
14cdef61 (HEAD -> master, upstream/master, origin/master, origin/HEAD) Merge pull request #2080 from mikeraynham/substate_guards
➜  xstate git:(master) ✗ node -v
v14.15.5
➜  xstate git:(master) ✗ yarn -v
1.21.1
```

**Comment by davidkpiano** [comment]
Work in progress here: https://github.com/davidkpiano/xstate/pull/2035

**Comment by davidkpiano** [comment]
This is now fixed by #2035
